nixos/modules/services/misc/sonarr.nix

```diff
@@ -54,7 +54,7 @@ in
         Type = "simple";
         User = cfg.user;
         Group = cfg.group;
-        ExecStart = "${cfg.package}/bin/NzbDrone -nobrowser -data='${cfg.dataDir}'";
+        ExecStart = "${lib.getExe cfg.package} -nobrowser -data=${lib.escapeShellArg cfg.dataDir}";
```

Comment:
I think the _technically correct_ way to do this is `utils.escapeSystemdExecArgs` (`utils` being a NixOS `specialArg`, like `lib`, `config`, or `pkgs`)

Reply:
Oh, thank you, I didn’t known this function exists!
```
$ cd nixpkgs/nixos
$ rg -l escapeShellArgs | wc -l
123
$ rg -l escapeSystemdExecArgs | wc -l
20
```
Doesn’t seem like it’s used in nixpkgs as much as it should be, unfortunately.